src/main/java/Duke.java

```diff
@@ -1,15 +1,46 @@
 
 import java.util.Scanner;
 
+class Task {
+    String taskName;
+    boolean status;
+
+    Task() {
+        this.status = false;
+    }
+
+    public String getTaskStatus() {
+        String status = " ";
+
+        if (this.status == true) {
+            status = "X";
+        }
+        return status;
+    }
+
+    public void mark() {
+        this.status = true;
+        System.out.println("目標達成！- [" + this.getTaskStatus() + "] " + this.taskName);
```

Comment:
Interesting way to differentiate Duke which is good. However, one possible way to make it user-friendly to all users would be to include the English translation